animations-tutorial.md

```diff
@@ -0,0 +1,638 @@
+---
+layout: page
+title: Animations Tutorial
+permalink: /animations-tutorial/
+---
+
+Welcome to Flutter's Animations Tutorial. You may also be interested
+in our [Animations Overview](/animations/).
```

Comment:
This sentence can be tightened: e.g.
"Before starting this tutorial, we encourage you to read the Animations overview documentation."